redhat-cop/casl-ansible
# Additional updates to the 'openshift_node_groups' inventory

#### What does this PR do?
Updated formatting for the 'openshift_node_groups` to use `yaml` formatting + fix indentation

#### How should this be manually tested?
Use the sample inventory to perform a provisioning of an OCP cluster. Observe that the ConfigMaps correct configured in the `openshift-node` project.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl

@raffaelespazzoli please see the "list formatting" in the OSP inventory. I left the GCP one as-is, but please let me know if you prefer that I update that one as well. (I'm fine leaving as-is as an examples of various ways to provide the configuration).

@makentenza please let me know if you have a preference for AWS. For now I've updated it with the yaml list formatting.
